package.json

```diff
@@ -21,13 +21,18 @@
     "dependencies": {
         "@babel/runtime": "^7.3.1",
         "apexcharts": "^2.6.0",
+        "axlsign": "^1.0.0",
+        "base-58": "0.0.1",
         "bignumber.js": "^7.2.1",
+        "blake2b": "^2.1.3",
         "bootstrap": "^4.1.1",
         "bootstrap-vue": "2.0.0-rc.28",
         "crypto-js": "^3.1.9-1",
         "identicon.js": "^2.3.2",
         "jr-qrcode": "^1.1.1",
+        "js-sha3": "^0.8.0",
         "json-bignumber": "^1.0.1",
+        "node-fetch": "^2.6.0",
         "vue": "^2.5.2",
         "vue-apexcharts": "^1.2.8",
         "vue-json-excel": "^0.2.5",
```

Comment:
add `"js-v-sdk": "./src/js-v-sdk"` here to install SDK dependencies. Or `qrcode` library will be missing.
![image](https://user-images.githubusercontent.com/41148850/65845721-ea19a380-e36d-11e9-842d-097100099f6a.png)